src/DataSheet.js

```diff
@@ -405,6 +411,18 @@ export default class DataSheet extends PureComponent {
       forceEdit: false
     })
 
+    var ua = window.navigator.userAgent
+    var isIE = /MSIE|Trident/.test(ua)
+    // Listen for Ctrl + V in case of IE
+    if (isIE) {
+      var thisContext = this
+        document.addEventListener('keydown', function (e){
```

Comment:
you can use lamda function here not to have an extra var here for passing context. Or bind the function